toolchain/sem_ir/formatter.cpp

```diff
@@ -1073,8 +1075,11 @@ class Formatter {
   auto FormatArg(ElementIndex index) -> void { out_ << index; }
 
   auto FormatArg(NameScopeId id) -> void {
-    out_ << '{';
-    FormatNameScope(id, ", ", "");
+    out_ << "{\n";
```

Comment:
Should this be inserting the `\n` even for an empty name scope?

Reply:
I have a follow-up PR to address this.